api/envoy/extensions/filters/http/lua/v3/lua.proto

```diff
@@ -78,6 +81,13 @@ message Lua {
   //         stat_prefix: bar_script # This emits lua.bar_script.errors etc.
   //
   string stat_prefix = 4;
+
+  // Determines if Envoy will exercise flow control when receiving HTTP body when Lua script executes httpCall.
+  // If the setting is true, Envoy will put backpressure on the downstream client to stop sending
+  // data. If the setting is false, Envoy will not put any backpressure on downstream client
+  // and will buffer HTTP body. Note that buffering may exceed maximum buffer limit per connection and the connection may be dropped.
+  // Defaults to false.
```

Comment:
I guess the main question I'd have API wise is whether this should just be the default behavior (with runtime control)?

Reply:
That new API flag is purely for backwards compatibility (if it is discovered that it somehow breaks executing Lua scripts). Assuming that is does not break anything, I cannot imagine why a user would choose one option vs the other. It seems that runtime control is a better option. @htuch do you suggest temporary (6 months long) runtime_guard?

Reply:
I'm good with a runtime guard providing folks who know more about the typical Lua use case than I don't think this is going to cause widespread breakage. Ideally we just have the right behavior by default.